.drone.yml

```diff
@@ -0,0 +1,96 @@
+kind: pipeline
+name: test
+
+steps:
+- name: docker build
+  image: docker:dind
+  volumes:
+  - name: dockersock
+    path: /var/run
+  commands:
+  - sleep 5; until docker version; do sleep 5; done
+  - docker build . --tag vagrant-vmck
+
+- name: list plugins
+  image: docker:dind
+  volumes:
+  - name: dockersock
+    path: /var/run
+  commands:
+  - apk update -q && apk add -q grep
+  - docker run vagrant-vmck vagrant --version
+  - docker run vagrant-vmck vagrant plugin list
+  - docker run vagrant-vmck vagrant plugin list | grep -q vagrant-vmck
+
+- name: launch example VM
+  image: docker:dind
+  volumes:
+  - name: dockersock
+    path: /var/run
+  commands:
+  - export VMCK_URL=http://$VMCK_IP:$VMCK_PORT
```

Comment:
Where do `$VMCK_IP` and `$VMCK_PORT` come from? Do we ever use them separately?

Reply:
You can't make drone put `:` in the environment variables that get passed to the jobs, because they are separated by `:`.

Reply:
Ah yes, wonderful.

Reply:
https://discourse.drone.io/t/setting-global-agent-environment-url-variables/4516 - maybe we should make a PR to them :)